src/analytics/private/metrics-common-workflow.ts

```diff
@@ -58,19 +74,18 @@ export function buildMetricsWidgetForWorkflows(scope: Construct, id: string, pro
     'service', MetricsService.WORKFLOW,
   ];
 
-  const loadEventsWorkflowAlarm = new Alarm(scope, id + 'loadEventsWorkflowAlarm', {
+  const loadEventsWorkflowAlarm = new Alarm(scope, id + 'loadEventWorkflowAlarm', {
     comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
     threshold: 1,
     evaluationPeriods: 1,
     treatMissingData: TreatMissingData.NOT_BREACHING,
-    metric: props.loadEventsWorkflow.metricFailed({ period: Duration.hours(1) }),
-    alarmDescription: `Load events workflow failed, projectId: ${props.projectId}`,
-    alarmName: getAlarmName(scope, props.projectId, 'Load events workflow'),
+    metric: props.loadDataWorkflows.event.metricFailed({ period: Duration.hours(1) }),
```

Comment:
Should the duration aligh with the interval of running data processing?

Reply:
the line-87, as below, overrides the hard-code value: 
```
  (loadEventsWorkflowAlarm.node.defaultChild as CfnResource).addPropertyOverride('Period', processingJobInterval.getIntervalSeconds());

```

cannot use the customer resource value directly in line-82, the hard-code value just a place-holder.

add comments to the hard-code lines.